packages/Pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Input from '@blaze-react/input';
+import usePagination from './hooks/usePagination';
+
+interface PaginationProps {
+  totalPages: number;
```

Comment:
Did you see this comment about changing `totalPages`? 

https://github.com/thebyte9/blaze-components-react/pull/628#discussion_r1410463926